### PR TITLE
GFORMS-2282: Automatically insert "languages": ["en","cy"] into form …

### PR DESCRIPTION
**Pull request by lxol**
…when inserting Welsh translations

### CODE REVIEW

**Review by DursunGundogan — APPROVED** [review]

File: app/uk/gov/hmrc/gform/translation/TranslationController.scala [hunk 1]
```diff
@@ -16,21 +16,35 @@
...
-import uk.gov.hmrc.gform.sharedmodel.formtemplate.{ FormTemplateId, FormTemplateRaw, FormTemplateRawId }
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.FormTemplateId
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.FormTemplateRaw
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.FormTemplateRawId
+
+import java.io.BufferedOutputStream
+import java.io.ByteArrayInputStream
+import java.io.ByteArrayOutputStream
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/translation/TranslationController.scala:41]: Why these import changes? This is not consistent with rest of code base?

Comment (lxol) [reply app/uk/gov/hmrc/gform/translation/TranslationController.scala:41]: `organize-import` in metals. Do we have a standard? Can we enforce it ?

Comment (josef-vlach) [reply app/uk/gov/hmrc/gform/translation/TranslationController.scala:41]: I bet you are alone to use `metals` on this team so it is quite controversial to do this. Can you reconfigure that functionality of not to use it?

Comment (lxol) [reply app/uk/gov/hmrc/gform/translation/TranslationController.scala:41]: ok, will do.

File: app/uk/gov/hmrc/gform/translation/TranslationController.scala [hunk 2]
```diff
@@ -90,6 +104,13 @@ class TranslationController(
       ) { csv =>
         formTemplateService
           .get(FormTemplateRawId(formTemplateId.value))
+          .map(r => FormTemplateRaw(r.value + ("languages" -> Json.toJson(Seq("en", "cy")))))
+          .flatMap(t =>
+            handler.handleRequest(t).value.flatMap {
+              case Right(_)              => Future.successful(t)
+              case Left(unexpectedState) => Future.failed(new RuntimeException(unexpectedState.error))
+            }
+          )
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/translation/TranslationController.scala:113]: Why are we inserting json twice, with two different implementation? Why not do it only once?
```
interpreter
              .handleRequest(FormTemplateRaw(jsonToSave))```

**Pull request merged**